### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: Landmark regions for Carousel 

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8914]
### Description

Page Element:
Landmark regions for Carousel 

Issue Description:
To provide more context to the screen reader user, the carousel which includes the slides and the Carousel controls could be given in a labelled landmark region.

Expected:
Provide the carousel including all the slides within a region role and aria label. E.g. role=”region” aria-label=”XYZ Carousel”

User Impact:
When the page content is not covered within a labelled landmark region, screen reader users are unable to skim through the page and understand its structure.

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Safari, Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by gopinathan-n** [comment]
@andy-blum @kennylam Issue is not fixed. The carousel section can be given within a labelled region role. This will help the user to understand the semantics of the carousel and also the beginning and ending boundary of the carousel section. Expected to provide the carousel section in region role with the aria-label="XYZ carousel" where "XYZ" is the name of the carousel

**Comment by Rasubra8** [comment]
Now the previous/next buttons are covered within a navigation region. Instead of this the entire carousel section starting from the H3 till the next button can be given within the region role with the aria-label="XYZ carousel.
